mappings/net/minecraft/item/FoodComponent.mapping

```diff
@@ -13,26 +14,58 @@ CLASS net/minecraft/class_4174 net/minecraft/item/FoodComponent
 		ARG 5 snack
 		ARG 6 statusEffects
 	METHOD method_19230 getHunger ()I
+		COMMENT Gets the amount of hunger a food item will fill.
+		COMMENT
+		COMMENT <p>One hunger is equivilant to half of a hunger bar icon.
+		COMMENT To fill a full hunger bar, this value must be at least 2.
 	METHOD method_19231 getSaturationModifier ()F
+		COMMENT Gets the staturation modifier of a food item.
+		COMMENT
+		COMMENT <p>This value is typically used to determine how long a player can sustain the current hunger value before the hunger is used.
 	METHOD method_19232 isMeat ()Z
+		COMMENT Checks if a food item can be fed to dogs.
 	METHOD method_19233 isAlwaysEdible ()Z
+		COMMENT Checks a food item can be eaten regardless of the current hunger value.
```

Comment:
```suggestion
		COMMENT Checks if a food item can be eaten when the current hunger bar is full.
```
Afaik in vanilla this limit is always "hunger value is full", and if there is no other restriction on hunger value we can just describe this specific case.

Ignore me if there can be other restrictions on hunger values aside from being not full.